trino/sqlalchemy/compiler.py

```diff
@@ -10,17 +10,8 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 from sqlalchemy.sql import compiler
-try:
-    from sqlalchemy.sql.expression import (
-        Alias,
-        CTE,
-        Subquery,
-    )
-except ImportError:
-    # For SQLAlchemy versions < 1.4, the CTE and Subquery classes did not explicitly exist
-    from sqlalchemy.sql.expression import Alias
-    CTE = type(None)
-    Subquery = type(None)
+from sqlalchemy.sql.base import DialectKWArgs
```

Comment:
Is it possible to add a test?

Reply:
I added a unit test with a TableClause instance.